packages/trader/src/Modules/Reports/Containers/open-positions.jsx

```diff
@@ -306,9 +305,6 @@ const OpenPositions = ({
 
         checkForMultiplierContract();
 
-        return () => {
-            if (!isMobile()) onUnmount();
-        };
         // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
What happens to the listeners in `onUnmount` method? there should be a way to dispose them!